apps/code-examples/src/app/code-examples/text-editor/text-editor-inline-help/text-editor-demo.module.ts

```diff
@@ -13,7 +12,6 @@ import { TextEditorDemoComponent } from './text-editor-demo.component';
     CommonModule,
     ReactiveFormsModule,
     SkyInputBoxModule,
-    SkyIdModule,
     SkyTextEditorModule,
     SkyHelpInlineModule,
```

Comment:
Can you remove the `SkyHelpInline` module?

Reply:
Fixed.